### DISCUSSION TITLE
Configuring TFS on GoCD

### DISCUSSION THREAD

**Discussion by SK702** [community/community#32891] [Other Feature Feedback, Questions, & Ideas]
Good day,

Please assist, I have installed GoCD in our environment, I am having issues to configure the URL Sites and repository.

**Comment by airtower-luna** [comment]
It looks like you're looking for help with a software hosted on GitHub, maybe https://github.com/gocd/gocd? In that case you should post your discussion or issue *in that repository* so the developers see it. :slightly_smiling_face:
